etc/kayobe/ansible/prometheus.yml.j2

```diff
@@ -0,0 +1,287 @@
+{{ '{%' }} raw {{ '%}' }}
```

Comment:
Sorry to crash the party - but that means we're going to maintain our version of prometheus.yml.j2 - and every time some new feature comes out in upstream land - we need to adapt it?

Reply:
In the near term we can switch to separate scrape configs to reduce the burden (supported from `2.43.0`). We just need to add support for it in KA (the binary was only updated the other day).

https://github.com/prometheus/prometheus/issues/8543

Reply:
Perhaps if kolla-ansible provided variables that allow you to define relabel configs for each scrape config that would allow us to avoid the custom template?

Reply:
I think we would need a high level of customisation to cover all use cases. 
To cover reuse of network devices for different purposes across different nodes, a new scrape_config for the different nodes would be needed. 

I'm not sure if seperate scrape_config files, possibly being templated similarly to haproxy services, would offer a meaningful alternative.

Reply:
A good idea @markgoddard - it would also be useful for CloudKitty. I have created https://stackhpc.atlassian.net/browse/INFRA-668